fetch.bs

```diff
@@ -4367,9 +4384,60 @@ steps:
   <p>Let <var>processResponseEndOfBody</var> be the following steps:
 
   <ol>
+   <li><p>Let <var>unsafeEndTime</var> be the <a>unsafe shared current time</a>.
+
    <li><p>Set <var>fetchParams</var>'s <a for="fetch params">request</a>'s
    <a for=request>done flag</a>.
 
+   <li>
+    <p>If <var>fetchParams</var>'s <a for="fetch params">request</a>'s
+    <a for=request>destination</a> is "<code>document</code>", then set <var>fetchParams</var>'s
+    <a for="fetch params">controller</a>'s <a for="fetch controller">full timing info</a> to
+    <var>fetchParams</var>'s <a for="fetch params">timing info</a>.
+
+    <p>Set <var>fetchParams</var>'s <a for="fetch params">controller</a>'s <a
```

Comment:
https://github.com/whatwg/fetch#formatting

"Using newlines between "inline" element tag names and their content is also forbidden. (This actually alters the content, by adding spaces.)"

Reply:
FYI the rule here is different between HTML, where such newlines are encouraged, and other WHATWG specs (at least @annevk-edited ones). Sorry about that :(

Reply:
No problem